content/en/docs/concepts/overview/components.md

```diff
@@ -24,7 +24,8 @@ a complete and working Kubernetes cluster.
 <!-- body -->
 ## Control Plane Components
 
-The control plane's components make global decisions about the cluster (for example, scheduling), as well as detecting and responding to cluster events (for example, starting up a new {{< glossary_tooltip text="pod" term_id="pod">}} when a deployment's `replicas` field is unsatisfied).
+The control plane's components make global decisions about the cluster (for example, scheduling), as well as detecting and responding to cluster events (for example, starting up a new {{< glossary_tooltip text="pod" term_id="pod">}} when a deployment's `{{< glossary_tooltip text="replicas" term_id="replica" >}}` field is unsatisfied
+).
```

Comment:
nit: This `(` can be moved to previous line to avoid unnecessary space during page render.
```suggestion
```